blog/2020/08/juliacon-2020-wrapup.md

```diff
@@ -0,0 +1,38 @@
+@def title = "JuliaCon 2020 Wrap-up"
+@def authors = "The JuliaCon 2020 Organizing Committee"
+@def published = "10 August 2020"
+@def rss_pubdate = Date(2020, 8, 10)
+@def rss = """JuliaCon 2020 Wrap-up"""
+
+JuliaCon 2020, the 7th annual Julia conference, has come to a close. This year, given the global COVID-19 Pandemic, the [organizing committee](https://juliacon.org/2020/committee.html) made the decision to move the conference from Lisbon Portugal into a [completely virtual experience](https://discourse.julialang.org/t/juliacon-2020-is-going-virtual/38327). This virtual format presented many challenges that we had to move quickly to solve. Avik and Jane (co-Chairs of JuliaCon) detailed some of the thinking and rationale for the high level design decisions that were made for this year's JuliaCon in the [final episode of the inaugural JuliaCon podcast](https://www.youtube.com/watch?v=hbHmH0fezjs). 
+
+Before we go further and summarize the outcomes of this year's JuliaCon, it is important to acknowledge the tremendous amount of work done by our organizing committee, moderators, speakers, and sponsors that enabled this year to be a success despite the new virtual format.
+
+## Outcomes
+
+JuliaCon 2020 had over 10,000 conference registrations this year with over 1,000 of those people joining us on the official conference discussion forum, discord. On top of the 10,000 registrations, we also garnered 26,000+ unique viewers on the Julia Languages YouTube channel during the conference (up from 10.8K unique viewers last year) and over 20,000 unique visitors to the JuliaCon website (a 140% growth from last year's conference). We are also pleased to report that according to Google Analytics, over 21% of the unique visitors were identified as female which may have potentially been as a result of concerted efforts to engage with and invite people from underrepresented groups in tech to attend JuliaCon. This effort will continue and grow into next year's conference with a focus on inviting diverse folks from specific technical backgrounds where Julia can benefit their work. 
```

Comment:
I checked the exact youtube analytics for unique users during the dates of JuliaCon 2019 and JuliaCon 2020. In 2019, it was 9,500 users, and this year it was 28,900 users.

Reply:
I widened the period by + & - 1 day to account for any delays in uploads given the conference was not online which is where the 10.8 figure is from.

Reply:
78 countries.